bugsnag-plugin-android-ndk/src/main/jni/bugsnag.c

```diff
@@ -118,7 +118,9 @@ void bugsnag_notify_env(JNIEnv *env, const char *name, const char *message,
   jbyteArray jname = NULL;
   jbyteArray jmessage = NULL;
 
-  if (!bsg_jni_cache_refresh(env)) {
+  bsg_jni_cache jni_cache;
+
+  if (!bsg_jni_cache_refresh(env, &jni_cache)) {
```

Comment:
This name doesn't really describe it properly anymore. It's not refreshing the cache, but rather filling it completely. Can we change it?